Fix #392: Docker setup results in 500 Internal Server Error

morevnaproject opened the pull request:
<!--
Thank you for contributing to UVDesk! Please fill out this description template to help us to process your pull request.
-->

### 1. Why is this change necessary?
After following instructions [here](https://github.com/uvdesk/community-skeleton/wiki/dockerize-helpdesk-project) user gets non-working Docker conatiner (500 Internal Server Error). Details explained here - https://github.com/uvdesk/community-skeleton/issues/392

### 2. What does this change do, exactly?
Added changes according to solution provided here - https://github.com/uvdesk/community-skeleton/issues/392#issuecomment-832982443

### 3. Please link to the relevant issues (if any).
https://github.com/uvdesk/community-skeleton/issues/392
https://github.com/uvdesk/community-skeleton/issues/229

papnoisanjeev commented:
@morevnaproject

Is composer install and composer update required as you have done in you pull request.
Already installing composer [here](https://github.com/uvdesk/community-skeleton/blob/master/Dockerfile#L62).

morevnaproject commented:
@papnoisanjeev The line you referenced is installing composer tool. And the lines added in this PR are using composer tool to install all dependencies required by UVDesk. ^__^



> Is composer install and composer update required as you have done in you pull request.

Yes. It doesn't works without them (both). You can check - just try to follow [official instructions](https://github.com/uvdesk/community-skeleton/wiki/dockerize-helpdesk-project) and you'll get 500 Internal Server Error. THis issue reported multiple times already (see #392 and #229).

papnoisanjeev commented:
@morevnaproject 

Probably it should work: https://github.com/uvdesk/community-skeleton/issues/392#issuecomment-832982443
without **composer update** as well.

morevnaproject commented:
> Probably it should work: [#392 (comment)](https://github.com/uvdesk/community-skeleton/issues/392#issuecomment-832982443)
> without **composer update** as well.

I have added `composer update` because it wasn't working without it. There was an error, which was advicing to run `composer update`. So I have added this line and it worked afterwards.

I believe this extra line is needed because of this commit - https://github.com/uvdesk/community-skeleton/commit/5de1cd16eb98eeb7d51001e5bb831cc68a817552

papnoisanjeev commented:
@morevnaproject 

>I believe this extra line is needed because of this commit - [5de1cd1](https://github.com/uvdesk/community-skeleton/commit/5de1cd16eb98eeb7d51001e5bb831cc68a817552)

Actually before it was restrict to use composer 1 only so after the above commit it should take latest composer (composer 2).

morevnaproject commented:
Yes, the error message was something about Composer 2 and requirement to run "composer update".

papnoisanjeev commented:
@morevnaproject 

Ok fine. 
Thanks for your contribution !!!
